src/GitHub/Listener/GitHubReleaseMastodonListener.php

```diff
@@ -34,9 +36,21 @@ public function __invoke(GitHubRelease $message): void
             return;
         }
 
+        $hashtags = ['#php'];
+
+        if (str_contains($message->getPackage(), 'mezzio')) {
```

Comment:
To be safe use  `str_starts_with(..., 'mezzio/')`

Reply:
For libs that contain both words, we end up with 3 hashtags - that's the intention, so `mezzio/mezzio-laminas-thing` gets `"#php #laminas #mezzio"`